Extensions/PrimitiveDrawing/Extension.cpp

```diff
@@ -379,6 +379,37 @@ void DeclarePrimitiveDrawingExtension(gd::PlatformExtension& extension) {
       .SetFunctionName("SetFillColor")
       .SetIncludeFile("PrimitiveDrawing/ShapePainterObject.h");
 
+  obj.AddExpression(
+        "GetFillColorR",
```

Comment:
Sorry I missed this earlier: there is never a `Get` for expressions, because they all are getters.
Also, let's avoid abbreviations: `FillColorRed`